app/src/main/java/com/github/wnder/Score.java

```diff
@@ -14,10 +17,15 @@
     //Calculate score from a distance only
     public static double calculationScore(double distance){
 
-        return calculationScore(distance, MAX_DISTANCE);
+        return calculationScore(distance, GlobalUser.getUser().getRadius() * 1000);
```

Comment:
It would probably be best to scrap this method and explicitly use the reference distance from the call site